### PR TITLE
fix docs and strings

### PR DESCRIPTION
**Pull request by stackoverflow**

### CODE REVIEW

File: grammar.js [hunk 1]
```diff
@@ -78,7 +78,6 @@ module.exports = grammar({
 
   extras: $ => [
     $.lineComment,
-    $.docComment,
```
Comment (stackoverflow) [inline grammar.js:81]: Doc comments shouldn't be a extra, but part of some nodes.

File: grammar.js [hunk 2]
```diff
@@ -523,19 +527,33 @@ module.exports = grammar({
...
-          $._sl6_string_chars,
+          alias($.slStringLiteralPart6, $.slStringLiteralPart),
           alias($.escapeSequence6, $.escapeSequence),
           alias($.interpolationExpr6, $.interpolationExpr)
         )),
         '"######'
       ),
     ),
 
+    slStringLiteralPart: $ => token.immediate(/[^"\\\n\r]+/),
```
Comment (stackoverflow) [inline grammar.js:538]: External rules are not generated as children in their parents, so I need to have normal rules here wrapping the external ones.

File: package.json [hunk 3]
```diff
@@ -27,15 +27,16 @@
...
     "ts-node": "^10.0.0",
     "typescript": "^4.9.0"
   },
   "scripts": {
     "build": "tree-sitter generate && node-gyp configure && node-gyp build",
     "test": "tree-sitter test",
     "synctests": "ts-node src/synctests.ts",
-    "getVersion": "node -p \"require('./package.json').version\""
+    "getVersion": "node -p \"require('./package.json').version\"",
+    "gen-lib": "tree-sitter generate -b --libdir ."
```
Comment (stackoverflow) [inline package.json:39]: This generates a shared library we need in the LSP.

**Review by holzensp — APPROVED** [review]
LGTM

**Pull request merged**